packages/client-api-schema/src/data-types.ts

```diff
@@ -114,6 +114,8 @@ export type ChannelStatus =
   | 'closing' // You cannot use the channel anymore, but your funds are still locked up.
   | 'closed'; // Your funds have been released from the channel.
 
+export type FundingStatus = 'Uncategorized' | 'ReadyToFund' | 'Funded' | 'Defunded';
```

Comment:
What does `'Uncategorized'` mean?

Reply:
It is a catch all label. So anything that cannot be categorized as `ReadyToFund`, `Funded` or `Defunded`